packages/cli-e2e/entrypoints/ci.sh

```diff
@@ -33,15 +33,20 @@ git config --global user.email "notgroot@coveo.com"
 echo "::group::Publishing UI templates"
 npm config set registry http://localhost:4873
 ts-node --transpile-only ./packages/cli-e2e/utils/npmLogin.ts
-yarn config set  registry http://localhost:4873
-yarn config set -- --mutex network
-yarn config set -- --install.silent true
-yarn config set -- --silent true
+# https://verdaccio.org/docs/cli-registry/#yarn-berry-2x
+echo 'npmRegistryServer: "http://localhost:4873"' >> .yarnrc.yml
+echo 'unsafeHttpWhitelist:"' >> .yarnrc.yml
+echo '  - localhost"' >> .yarnrc.yml
+
+touch ~/.yarnrc.yml
+echo 'npmRegistryServer: "http://localhost:4873"' >> ~/.yarnrc.yml
+echo 'unsafeHttpWhitelist:"' >> ~/.yarnrc.yml
+echo '  - localhost"' >> ~/.yarnrc.yml
+
```

Comment:
Advice 'let it be' for now, all CI specifics for verdaccio are removed in a upcoming PR.
This is purely transitory